newt/vendor/mynewt.apache.org/newt/util/util.go

```diff
@@ -42,6 +42,7 @@ import (
 )
 
 var Verbosity int
+var PrintBuildCmds bool
```

Comment:
Since the util library is not just used by newt (e.g., it is used by newtmgr), I would rename this variable to something more generic such as `PrintShellCmds`.

Also, the changes look good to me, but the changes to util.go were only made to the vendored version of this file.  The Go vendoring system is kind of messy, and it creates a lot of hassle when all you want to do is make a small change.

Basically, there are several versions of util.go:
* util/util.go
* newt/vendor/mynewt.apache.org/newt/util/util.go
* newtmgr/vendor/mynewt.apache.org/newt/util/util.go

The first one (util/util.go) is the "master".  The others are just copies that get distributed with the tool that depends on it.

The correct way to update util.go is to make your changes to util/util.go, and then *revendor* the tools which require the updated util package.  This involves running the `godep` tool, and can be a bit tricky to do when everything is in the same repo, as in this case.  It is important to make sure your changes go into util/util.go, or else they will be lost the next time someone revendors newt.